guide/getting-started/technology-primer.md

```diff
@@ -73,264 +79,265 @@ It's both. Lowercase bitcoin refers to the currency. So you may have 1 bitcoin i
+Imagine a global spreadsheet where all transactions get recorded, and all entries are publicly accessible. With Bitcoin, each participant can verify a transaction themself in their copy of the spreadsheet. In comparison, traditional banks keep their spreadsheets private from one another, including their customers. Besides opportunities for fraud, there are many points of failure due to this, so you can't guarantee a payment will always happen. Even though all transactions are stored publicly on Bitcoin, there is no personally identifiable information attached to them.
 
-This makes backing up your wallet fairly easy, as you only need to store your recovery phrase to restore your wallet securely. However, it also concentrates the risk of having your bitcoin stolen or lost to this one piece of information. Thus, you supporting and guiding users in backing up and securing their wallets are among the most fundamental goals for designers.
+With thousands of nodes and tens of millions of users worldwide, it also becomes implausible to manipulate or corrupt the previously agreed-upon transactions stored in the blockchain. To do so would require a majority of nodes to accept such change.
 
+**More info**
+- [Explorer software]({{ '/guide/getting-started/software/#explorers' | relative_url }})
 </div>
 
-## How do you control an address?
+## What is a blockchain?
 
 <div class="center" markdown="1">
 
 {% include image.html
-   image = "/assets/images/guide/getting-started/technology/address-and-private-key.jpg"
-   retina = "/assets/images/guide/getting-started/technology/address-and-private-key@2x.jpg"
-   alt-text = "Combination of an address and a private key"
+   image = "/assets/images/guide/getting-started/technology/blockchain.jpg"
+   retina = "/assets/images/guide/getting-started/technology/blockchain@2x.jpg"
+   alt-text = "Several blocks chained together via their hashes"
    width = 400
    height = 400
    layout = "float-right-desktop"
 %}
 
-As we now know, bitcoins are just numbers linked to addresses. An address can receive bitcoin, but to spend that bitcoin, a private key is needed. Wallets create private keys first and then derive matching addresses from them. While an address can be publicly shared, the private key should be kept a secret.
-
-Wallet software typically hides private keys from users, as there is no need to interact with them directly.
+Instead of processing each transaction individually, the Bitcoin network bundles them into [blocks]({{'/guide/glossary/#hash' | relative_url }}). A new block is created roughly every 10 minutes and can only contain a certain amount of transactions due to a strict file size limit, so transactions with the highest fees typically get chosen first. The new block must link to the one before it. Since this is a continuous process, a chain of blocks forms, hence the name blockchain.
```

Comment:
```suggestion
Instead of processing each transaction individually, the Bitcoin network bundles them into [blocks]({{'/guide/glossary/#hash' | relative_url }}). A new block is created roughly every 10 minutes. A strict file size limit results in the transactions with the highest fees typically getting chosen first. Every new block must link to the one before it. Since this is a continuous process, a chain of blocks forms, hence the name blockchain.
```

`Blocks` links to `/guide/glossary/#hash`, which does not seem right. Where should it link to?